.github/workflows/format_check.yml

```diff
@@ -0,0 +1,41 @@
+name: format-check
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release-'
```

Comment:
potentially not relevant to our workflows?

Reply:
yeah, I'm not sure. I've switched it to the regex that's in `ci.yml` so at least it's consistent across workflows. I've used release branches with Convex.jl once or twice when I want to backport a fix to a previous version, so I think it's reasonable to at least have CI that understands that use-case, even if it's rare.